internal/rooms/roommanager.go

```diff
@@ -307,6 +310,60 @@ func MoveToRoom(userId int, toRoomId int, isSpawn ...bool) error {
 		}
 	}
 
+	// Send PreRoomChange event before room description
```

Comment:
Technically itd oesn't matter when we queue the send, since they go into different queues, and only the order the queues process matters. This does give me some ideas though.